lib/reducers/create-otp-reducer.js

```diff
@@ -165,6 +165,8 @@ export function getInitialState(userDefinedConfig) {
 
   const sessionId = new URLSearchParams(getUrlParams()).get('sessionId')
 
+  const nearbyFilters = config.nearbyView.filters
```

Comment:
```suggestion
  const nearbyFilters = config?.nearbyView?.filters
```
Shouldn't this stay optional!

Reply:
Fix it crashing and made it crash again in the same commit... my mind 🚀